content/blog/en/version-6-dojo.md

```diff
@@ -0,0 +1,295 @@
+
+To build your widgets using `@dojo/cli-build-widget`, list your widgets in the `widgets` section of the `.dojorc` and run the build using the `--lib` option. The resulting build output is ready to be packaged and consumed by other Dojo applications.
+
+> .dojorc
+```json
+{
+  "build-widget": {
+    "widgets": [
+      "src/MyWidget"
+    ]
+  }
+}
+```
+
+> terminal
+```shell
+dojo build widget --lib
+```
+
+## Faster Development Builds
+
+As projects grow in size their build times can significantly increase. The CLI build command now supports an experimental “fast” mode that can reduce the build time of larger projects while developing.
+
+![experimental speed demo](assets/blog/version-6-dojo/speed.gif)
+
+This shows an approximate saving of over 2 seconds building the [Dojo RealWorld example](https://github.com/dojo/examples/tree/master/realworld) with the experimental speed mode enabled. On larger Dojo projects we've witnessed more significant savings with development build times dropping by more than 50% having previously taking over 40 seconds, reducing to under 20 seconds using the new speed mode.
+
+## Glob Support For Code Splitting
+
+The `.dojorc` configuration for `bundles` has been enhanced to support globs. This is especially useful for scenarios such as internationalization, meaning that you don't have to define all the language bundles explicitly, instead simply define a matching pattern for each of the locales.
+
+Consider a project with a folder structure that defines language bundles in locale named directories, ideally build tool would create a single bundle for each locale that would be loaded when users change locale. Using a glob provides a low maintenance and minimal effort way to do this over explicitly defining each language bundle modules in the `.dojorc`.
+
+Current configuration required to create a bundle per locale:
+
+> .dojorc
+```json
+{
+  "build-app": {
+    "bundles": {
+      "fr": [ 
+        "src/widgets/home/nls/fr/home",
+        "src/widgets/menu/nls/fr/menu",
+        "src/widgets/blog/nls/fr/blog",
+        "src/widgets/reference/nls/fr/reference"
+      ],
+      "jp": [ 
+        "src/widgets/home/nls/jp/home",
+        "src/widgets/menu/nls/jp/menu",
+        "src/widgets/blog/nls/jp/blog",
+        "src/widgets/reference/nls/jp/reference"
+      ]
+    }
+  }
+}
+```
+
+Using the glob configuration to create a bundle per locale:
+
+> .dojorc
+```json
+{
+  "build-app": {
+    "bundles": {
+      "fr": [ "**/fr/*" ],
+      "jp": [ "**/jp/*" ]
+    }
+  }
+}
+```
+
+## Update to TypeScript
+
+The minimum required TypeScript version has been updated to 3.4. Updating the core framework to use a recent version enables us to leverage the latest features that underpin the function-based widget and middleware typings.
```

Comment:
Should we mention that this is the first time we've changed this in over a year and give some guidance that we similarly don't plan to increment the minimum version for about the same amount of time unless absolutely necessary?

Reply:
I don't think so personally.

Reply:
ok

Reply:
dojo/framework's readme says the minimum is `3.4.5` versus this implies you could use `3.4.0`.